implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md

```diff
@@ -187,9 +190,98 @@ export default ({ facet }) => {
     </div>
 }
 ```
-{% endcode %}
 
-Utilize the `useRangeSlider` hook to generate useful context for rendering range inputs. Additionally, employ the component to generate the slider itself. These tools together facilitate the creation of dynamic and interactive range sliders for your application.
+##### Range Selector
+
+If you require an alternative method where values are selected through radio buttons rather than a slider, consider using `useRangeSelector` hook. This tool allows users to choose from predefined range intervals with radio buttons, offering a different interaction style that may be preferable in certain contexts. 
+The range size parameter in `useRangeSelector` hook, such as 100, sets the number of intervals between the min and max values. 
```

Comment:
The range size parameter in `useRangeSelector` hook can be used to specify the size of each items in the range. It also determines the total number of range items displayed. For example, 
a. if min product price in the catalog, based on filter, is 200
b. if max product price in the catalog, based on filter, is 2000
a range size of 200 will have the following items displayed under the "Price" filter

- 200 - 400
- 400 - 600
- 600 - 800
- 800 - 1000
- 1000 - 1200
- 1200 - 1400
- 1400 - 1600
- 1600 - 1800
- 1800 - 2000